rinja/src/dynamic/child.rs

```diff
@@ -0,0 +1,207 @@
+use std::io::ErrorKind;
+use std::process::exit;
+use std::string::String;
+use std::sync::Arc;
+use std::time::Duration;
+use std::vec::Vec;
+use std::{eprintln, format};
+
+use linkme::distributed_slice;
+use tokio::io::{AsyncBufReadExt, AsyncWriteExt, BufReader, Stdin, Stdout, stdin, stdout};
+use tokio::spawn;
+use tokio::sync::{Mutex, oneshot};
+
+use super::{DYNAMIC_ENVIRON_KEY, MainRequest, MainResponse, Outcome};
+
+const PROCESSORS: usize = 4;
+
+#[inline(never)]
+pub(crate) fn run_dynamic_main() {
+    std::env::set_var(DYNAMIC_ENVIRON_KEY, "-");
+
+    let mut entries: Vec<_> = DYNAMIC_TEMPLATES.iter().map(|entry| entry.name()).collect();
+    entries.sort_unstable();
+    eprintln!("templates implemented by subprocess: {entries:?}");
```

Comment:
`println!` instead maybe? It's not an error after all.

Reply:
STDIN & STDOUT are reserved from communication with the parent process.

Reply:
Sounds like a code comment is needed. :)

Reply:
Yeah, possibly even a small UML'like actor diagram.